composer.json

```diff
@@ -26,7 +26,7 @@
     "issues": "https://github.com/phootwork/phootwork/issues"
   },
   "require": {
-    "php": ">=7.2",
+    "php": ">=7.2 || 8.0.*",
```

Comment:
This has been problematic version semver setting. And the Composer presents following errors:

```PHP
[UnexpectedValueException]                                                  
  Could not parse version constraint  8.0.*: Invalid version string " 8.0.*"  
```

Reply:
How about setting `^8.0`?